exonum-java-binding-cryptocurrency-demo/frontend/.gitignore

```diff
@@ -2,3 +2,6 @@ node_modules/
 npm-debug.log*
 /dist/
 /build/
+
+# The directory in which npm & node are downloaded by the Maven frontend plugin.
+/node/
```

Comment:
That's the default (as you see, it is not in the target, but in the `frontend` directory). As a :heavy_plus_sign: , `mvn clean` does not touch it, therefore, local builds won't download the npm & nodejs after each of your clean.